src/helpers/private.route.js

```diff
@@ -2,17 +2,17 @@ import React from "react";
 import { useSelector } from "react-redux";
 import { Redirect, Route } from "react-router-dom";
 
-const PrivateRoute = ({ component: Component, ...rest }) => {
+const PrivateRoute = ({ comp: Component, ...rest }) => {
```

Comment:
Why this change? just being curious

Reply:
Because a warning was thrown with Private route, and I tried something I saw in the bible (stackoverflow). And it worked haha